src/Surfnet/StepupMiddleware/ApiBundle/Identity/Repository/InstitutionListingRepository.php

```diff
@@ -142,7 +142,7 @@ public function getInstitutionsForSelectRaaAsSraa(Institution $institution)
                 InstitutionAuthorization::class,
                 'a',
                 Join::WITH,
-                "r.institution = a.institution AND a.institutionRole IN (:authorizationRoles)"
+                "i.institution = a.institution AND a.institutionRole IN (:authorizationRoles)"
```

Comment:
Didn't you bugfix this yesterday? Was that fix not correct? Or am I missing something here?

Reply:
Yes, I thought I needed to fix this but this is not going to work for the query for SRAA's because we don;t have an alias `r`.

Reply:
Acknowledged!